app/views/users/new.html.erb

```diff
@@ -1,3 +1,24 @@
 <% provide :title, t(".signup") %>
 <h1><%= t ".signup" %></h1>
 <p><%= t ".detail" %></p>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user, url: signup_path) do |f| %>
```

Comment:
form_for là gì nhỉ, em thử inspect trình duyệt lên xem các thẻ của nó là gì rồi giải thích giúp anh nhé :D

Reply:
khi inspect thì form for đấy chạy ra thẻ form. Mỗi  phương thức f. được gọi tương ứng với 1 thẻ html. Theo em hiểu thì thẻ form_for được Rail xây dựng để check thông tin do người dùng nhập vào, cho phép người dùng tạo và sửa thông tin